client/src/components/SideBar.tsx

```diff
@@ -46,8 +46,8 @@ const navLinks = [
     icon: <Bed />,
   },
   {
-    title: 'Users',
-    path: '/users',
+    title: 'Admins',
+    path: '/Admins',
```

Comment:
Thank you so much everything looks good to me, except the path should be all lowercase.